tasks/configure_agent.yml

```diff
@@ -16,6 +16,16 @@
     - nrinfragent_state != "absent"
     - nrinfragent_os_name != 'windows'
     - ansible_python_version is version('3.0', '>=')
+    - ansible_facts['os_family'] == 'Debian'
```

Comment:
i think this will create more issues cause every package manager doesn't have python3-yaml. as an example SUSE family has "python3-PyYAML" so it will break the condition, if we can stick with family blocks that will be much clear